From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Removed SVG and wikidraw from wikiconfig.

The drawing-related modules are not yet packaged and should be optional,
anyway, so this patch comments them out in the wikiconfig.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.

Last-Updated: 2024-02-23
Reviwed-By: Kathara Sasikumar <katharasasikumar007@gmail.com>

Index: moin/src/moin/config/wikiconfig.py
===================================================================
--- moin.orig/src/moin/config/wikiconfig.py
+++ moin/src/moin/config/wikiconfig.py
@@ -227,7 +227,7 @@ class Config(DefaultConfig):
         'font_awesome',
         'ckeditor',
         'autosize',
-        'svgedit_moin',
+       #'svgedit_moin',
         'jquery_tablesorter',
         'pygments',
     ]
